pyproject.toml

```diff
@@ -28,6 +28,7 @@ click = "^8.1.7"
 python-dotenv = "1.0.0"
 embedchain = "^0.1.98"
 appdirs = "^1.4.4"
+agentops = "^0.1.1"
```

Comment:
Let's make it optional, same way we do to crewai-tools, so we don't add the extra dependency for everyone, feel unnecessary if people are not using agent ops